### DISCUSSION TITLE
How to make the text in the center in the button grid

### DISCUSSION THREAD

**Discussion by zhangguanghuib** [microsoft/Dynamics365Commerce.Solutions#213] [Q&A] [answered]
Hello team,

Customer want to make the text in the center in the button grid on POS like that:
<img width="133" alt="image" src="https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/14832260/775fb05d-5110-449e-a131-91447adb7573">

But the default behavior is like:
<img width="119" alt="image" src="https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/14832260/0aa60327-7ca5-4641-aacb-71692e7da496">

As per my investigation, it is controlled by this code:
<img width="1309" alt="image" src="https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/14832260/89d8f73e-8f03-4831-84be-be5037a28f66">

Question:
Is it possible to override this style to make the text in the center of the button?

**Comment by madyke** [comment, marked as answer]
This was addressed with support internally, but responding here in case other customers are interested:

As you noted, the button grid button text is hard coded to be left aligned, and unfortunately there is no way to modify this behavior to center the button text. 

If the customer would like to add this as a configuration option, I would recommend creating a new idea in the ideas portal and if enough customers are interested in this behavior, we can add it to our backlog.
